bindata/manifests/metrics-exporter/metrics-service.yaml

```diff
@@ -18,6 +18,7 @@ spec:
       name: sriov-network-metrics
       port: {{ .MetricsExporterPort }}
       targetPort: {{ .MetricsExporterPort }}
+{{ if .IsOpenshift }}
```

Comment:
I think here we can also support k8s

lets check if the ServiceMonitor CRD exist in the cluster and deploy it instead of checking only for openshift WDYT?

Reply:
good idea, working on that

Reply:
Done, I used the unstructured client to check if the ServiceMonitore resource definition is available in the cluster. Does it sound good?